coalib/misc/Shell.py

```diff
@@ -52,6 +54,11 @@ def run_interactive_shell_command(command, **kwargs):
     """
     if not kwargs.get('shell', False) and isinstance(command, str):
         command = shlex.split(command)
+    else:
+        command = list(command)
+
+    if sys.platform.startswith('win'):  # pragma: no cover
```

Comment:
I think you should place a comment above this stmt explaining why we do this
\+
Better use `if platform.system() == 'Windows'` :+1: